tests/integration/models/application.spec.ts

```diff
@@ -118,15 +118,13 @@ describe('Application Model', function () {
 					);
 				});
 
-				it('should be rejected if the device type is discontinued', function () {
+				it('should succeed even if the device type is discontinued', function () {
```

Comment:
This should no longer be in the `[read operations]` section since it actually manages to create a fleet. We need to move it to the `[mutating operations]`.